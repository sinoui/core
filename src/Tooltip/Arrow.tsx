import styled from 'styled-components';

const Arrow = styled.div`
  z-index: -1;
  position: absolute;
  &::before {
    width: 10px;
    height: 10px;
    position: absolute;
  }
  &::before {
    content: '';
    transform: rotate(45deg);
    background-color: rgb(97, 97, 97);
    top: 0;
    left: 0;
  }
`;

export default Arrow;
