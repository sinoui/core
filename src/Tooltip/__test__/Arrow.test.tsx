import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import { defaultTheme } from '@sinoui/theme';
import '@testing-library/jest-dom';
import 'jest-styled-components';
import Arrow from '../Arrow';

it('ToolTipContent', () => {
  const tree = renderer.create(
    <ThemeProvider theme={defaultTheme}>
      <Arrow />
    </ThemeProvider>,
  ).toJSON;

  expect(tree).toMatchSnapshot();
});
