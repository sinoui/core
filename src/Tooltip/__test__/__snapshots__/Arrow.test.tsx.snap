// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolTipContent 1`] = `[Function]`;
